commands/user_test.go

```diff
@@ -52,3 +52,302 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
 		s.Require().Equal("Unable to find user 'example@example.com'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestUserInviteCmd() {
+	s.Run("Invite user to an existing team by Id", func() {
+		printer.Clean()
+		argUser := "example@example.com"
+		argTeam := "teamId"
+
+		s.client.
+			EXPECT().
+			GetTeam(argTeam, "").
+			Return(&model.Team{Id: "teamId"}, &model.Response{Error: nil}).
```

Comment:
nit: use the already created variable `argTeam` to relate it with the expected team

```suggestion
			Return(&model.Team{Id: argTeam}, &model.Response{Error: nil}).
```